charts/vault/values/plugin-secrets-kubernetes-reader.yaml

```diff
@@ -0,0 +1,34 @@
+server:
+  dev:
+    enabled: true
+  extraArgs: '-dev-root-token-id=root -dev-plugin-dir=/usr/local/libexec/vault'
+  volumes:
+    - name: plugins
+      emptyDir: {}
+  volumeMounts:
+    - mountPath: /usr/local/libexec/vault
+      name: plugins
+      readOnly: false
+  extraInitContainers:
+    - name: kubesecrets-plugin
+      image: ghcr.io/the-mesh-for-data/vault-plugin-secrets-kubernetes-reader:latest
+      imagePullPolicy: Always
```

Comment:
If this is omitted then it's also omitted from the deployment? I think that would be a better default.

Reply:
Please explain not sure I understand... tnx

Reply:
Suggestion is to omit `imagePullPolicy: Always` and keep default behavior (IfNotExists unless tag is latest)